### PR TITLE
Report uncaught exceptions with sentry.io

### PR DESCRIPTION
**Pull request by justinabrahms**
We still need to upload debug symbols for release builds.

### CODE REVIEW

File: xcode/Subconscious/Subconscious.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1562,6 +1567,28 @@
...
+/* Begin PBXShellScriptBuildPhase section */
+		8804D2562A424FD900D45E83 /* Run Script */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+				"${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Resources/DWARF/${TARGET_NAME}",
```
Comment (bfollington) [inline xcode/Subconscious/Subconscious.xcodeproj/project.pbxproj:1579]: image

File: xcode/Subconscious/Subconscious.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1562,6 +1567,28 @@
...
+			);
+			name = "Run Script";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "if which sentry-cli >/dev/null; then\nexport SENTRY_ORG=subconscious\nexport SENTRY_PROJECT=apple-ios\nexport SENTRY_AUTH_TOKEN=cfb08a61febc418bb55ec80ae6f99e2ce34b93d5b179454e9a8b8c240a3bb4d6\nERROR=$(sentry-cli debug-files upload \"$DWARF_DSYM_FOLDER_PATH\" 2>&1 >/dev/null)\nif [ ! $? -eq 0 ]; then\necho \"warning: sentry-cli - $ERROR\"\nfi\nelse\necho \"warning: sentry-cli not installed. download from https://github.com/getsentry/sentry-cli/releases\"\nfi\n";
```
Comment (bfollington) [inline xcode/Subconscious/Subconscious.xcodeproj/project.pbxproj:1588]: Is it okay for this auth token to be publicly checked in? Bearing in mind this repo is fully open-source.

This issue seems to suggest we shouldn't check it in: https://github.com/getsentry/sentry-react-native/issues/1096

**Review by bfollington — COMMENTED** [review]
I want to be sure that we're handling the auth token appropriately before merging.

**Comment by justinabrahms** [comment]
That token has been revoked. Good catch. I'll work w/ you in DM to figure out how to pass in relevant secrets.

**Review by bfollington — APPROVED** [review]
LGTM! Thanks for tackling the .env, that was well overdue.

**Pull request merged**